Tests/Editor/CoreApi/GameLiftComputeManagerTests.cs

```diff
@@ -82,9 +82,9 @@ public void RegisterFleetCompute_WhenCorrectInputs_ExpectSuccess()
             
             //Assert
             _gameLiftWrapperMock.Verify(wrapper => wrapper.RegisterCompute(It.IsAny<RegisterComputeRequest>()), Times.Once);
-            _coreApiMock.Verify(f => f.PutSetting(It.IsAny<string>(), It.IsAny<string>()), Times.Exactly(3));
+            _coreApiMock.Verify(f => f.PutSetting(It.IsAny<SettingsKeys>(), It.IsAny<string>()), Times.Exactly(3));
             
-            Assert.IsTrue(registerComputeResponse.Success);
+            Assert.IsTrue(registerComputeResponse);
```

Comment:
Shouldn't this monitor the Success/Failure of the Response? Wondering if this condition is incorrect since this will be true if the Response itself exists? I'm not sure how C# deals with that

Reply:
Same for the other Assertions in this file

Reply:
The result was changed to a bool, not Response<bool> but the tests were not changed

Reply:
Why shouldn't this return a Response? From which a Success/Failure can be inferred rather than a boolean? That's typically how all APIs should be designed. This is a follow up comment which can happen as part of a later PR.